ci/environment.yml

```diff
@@ -24,3 +24,4 @@ dependencies:
   - zarr
   - pip:
       - -r ../requirements.txt
+      - git+https://github.com/NCAR/xcollection
```

Comment:
I just released a [new version.](https://github.com/NCAR/xcollection/releases/tag/v2021.12.01) So, we should be able to remove this

```suggestion
```